core/src/apps/common/sd_salt.py

```diff
@@ -36,15 +26,10 @@ class SdProtectCancelled(Exception):
         btn_confirm = None
         btn_cancel = "Close"
 
-    if ctx is None:
-        if await Confirm(text, confirm=btn_confirm, cancel=btn_cancel) is not CONFIRMED:
-            raise SdProtectCancelled
-    else:
-        if not await confirm(ctx, text, confirm=btn_confirm, cancel=btn_cancel):
-            raise wire.ProcessError("Wrong SD card.")
```

Comment:
It looks like all of these wire.ProcessErrors were lost in the refactor. So won't that produce unhandled exceptions, which will show up as firmware errors on the host?